utils/feed_fetcher.py

```diff
@@ -34,7 +34,7 @@
 from django.utils.encoding import smart_unicode
 from utils import json_functions as json
 from celery.exceptions import SoftTimeLimitExceeded
-from vendor import tweepy
```

Comment:
Aww geez, you're absolutely right. Thanks for catching this!
<img width="710" alt="screen shot 2016-09-16 at sep 16 2 39 11 pm" src="https://cloud.githubusercontent.com/assets/44229/18602042/5efcf8cc-7c1b-11e6-8a5f-dcf07683847f.png">